pkg/ipmi/ipmi_linux.go

```diff
@@ -18,5 +18,5 @@ func Open(devnum int) (*IPMI, error) {
 		return nil, err
```

Comment:
I would try ipmi/linux.go and if that does not work to restrict its use, add a build tag.